packages/instantsearch-react/src/impl/Range.js

```diff
@@ -0,0 +1,50 @@
+import React, {PropTypes, Component} from 'react';
+import Slider from 'rc-slider';
+
+// @TODO: rc-slider doesn't support our customization options relative to
+// theming. We'll need to implement our own slider down the road.
+export default class Range extends Component {
+  static propTypes = {
+    min: PropTypes.number.isRequired,
+    max: PropTypes.number.isRequired,
+    value: PropTypes.arrayOf(PropTypes.number).isRequired,
```

Comment:
I like the singular `value`. `values` implies that all items inside have the same role, while in reality the format of the prop is `[min, max]`.